sql/src/main/java/io/crate/analyze/relations/select/SelectItemAnalysis.java

```diff
@@ -36,7 +36,7 @@
 import java.util.List;
 import java.util.Map;
 
-public class SelectAnalysis {
+public class SelectItemAnalysis {
```

Comment:
I'd avoid class renames on feature branches that might live a bit longer as it increases potential for conflicts.
Or at least keep it in a separate commit as it causes a lot of noise and you're then putting the burden on the reviewer to filter out what is a "real change" and what is simply a naming change.

Also not sure if it is an improvement. This components are taking care of the full `SELECT` clause, not just of individual items within the clause.